### PR TITLE
feat: read config json from documents dir

### PR DESCRIPTION
**Pull request by Saschl**
Needs https://github.com/flybywiresim/simbridge/pull/121 to be released first.

## Additional context

Discord username (if different from GitHub):

### CODE REVIEW

File: src/renderer/utils/InstallerConfiguration.ts [hunk 1]
```diff
@@ -354,7 +354,7 @@ export interface Configuration {
 
 export class InstallerConfiguration {
   static async obtain(): Promise<Configuration> {
-    return this.fetchConfigurationFromCdn()
+    return this.loadConfigurationFromLocalStorage()
```
Comment (FoxtrotSierra6829) [inline src/renderer/utils/InstallerConfiguration.ts:357]: please wait for #477 to implement a developer option to force the use of a local configuration or use https://github.com/flybywiresim/installer-data/pull/40 for testing.

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]

File: src/renderer/utils/InstallerConfiguration.ts (hunk 1 above)
Comment (Saschl) [reply src/renderer/utils/InstallerConfiguration.ts:357]: Whoops that change wasn’t meant to be committed, will revert.

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]
grafik
In the about page, the html parsing is not implemented. Also, the fallback is missing for the resources folder which would be helpful to access to move the files before updating.

**Comment by Saschl** [comment]
> grafik In the about page, the html parsing is not implemented. Also, the fallback is missing for the resources folder which would be helpful to access to move the files before updating.

Fixed the first, thanks 👍  About the fallback: we could also split the installer-data PR into two.

1. Only update the description to raise awareness -> installer works as before 
2. Update the path together with the simbridge release -> installer will open the new directory with the resources button

**Comment by Saschl** [comment]
> Open for discussion, I'd rather prefer a solution that is not strictly time-based since not everyone uses the FlightSim on a daily basis

Yep, I see the point. I just want to keep the change as minimal as possible. Please check again, I've added another directories entries in the config which points to the old one. For me it's a good compromise :D 

Otherwise I would think it's too much. If we go with my original proposal, only people who ignore the update will have an issue with the "Resources" button and the configuration in the installer. I would consider that acceptable.

**Review by FoxtrotSierra6829 — APPROVED** [review]

**Pull request merged**